src/utilities/inputProcessors.ts

```diff
@@ -149,6 +150,20 @@ export const processInputReleaseBranches = (): BranchSpec[] | undefined => {
   return validateInputReleaseBranches(parsedInput);
 };
 
+export const processInputConfigFile = (): string | undefined => {
+  const file = parseFileList(getInput(InputParameters.ConfigFile)).shift();
```

Comment:
Shift modifies the reference array. Are you very sure that it is safe to do that?

Reply:
Yes :)